app/models/supporting_evidence.rb

```diff
@@ -0,0 +1,13 @@
+class SupportingEvidence < ApplicationRecord
+  self.table_name = 'supporting_evidence'
+  belongs_to :claim
+
+  attribute :file_name
+  attribute :file_type
+  attribute :file_size
+  attribute :case_id
+  attribute :id
+  attribute :send_by_post
```

Comment:
- send_by_post is hangover
- I would rather specify the attributes to avoid the "magic" - Its easy to see what the model shape is this way

Reply:
Not 100% against the attributes to avoid magic... but unless we are doing that everywhere I would push back... also very not rails approach to it should should be documented why we did this and ensure it is maintained in the future.